### ISSUE TITLE
Funding an event

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#604]
It has occurred to me that the invoice system being built as part of #505 could be used to facilitate the reimbursement of travel expenses by the organizers of a conference. In that case it would be logical to also allow collecting funds for the event on Liberapay, thus implementing something like #603 but for completely non-recurrent funding.

Opinions?

**Comment by Zatalyz** [comment]
Good idea, like #603 . Allow for the financing of a specific project or event is very useful, with or without time limit. But how to include it ? Event are they related to a person, a group ? Who have the money ?
